.eslintignore

```diff
@@ -1,2 +1,2 @@
-npm node_modules
```

Comment:
I think this was wrong, there is no `build` directory and `npm node_modules` doesn't make sense